tonic-build/src/client.rs

```diff
@@ -59,21 +58,19 @@ pub fn generate<T: Service>(
                     #service_ident::new(InterceptedService::new(inner, interceptor))
                 }
 
-                #methods
-            }
-
-            impl<T: Clone> Clone for #service_ident<T> {
-                fn clone(&self) -> Self {
-                    Self {
-                        inner: self.inner.clone(),
-                    }
+                // TODO(david): docs
+                pub fn send_gzip(mut self) -> Self {
+                    self.inner = self.inner.send_gzip();
+                    self
                 }
-            }
 
-            impl<T> std::fmt::Debug for #service_ident<T> {
-                fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-                   write!(f, #struct_debug)
```

Comment:
`tonic::client::Grpc` actually contains useful data now. So figured it makes sense to change this `Clone` and `Debug` impl to derives. Made sure the `Derive` doesn't require `T: Derive`.